src/protocol.cpp

```diff
@@ -628,6 +628,14 @@ if ( rand() < ( RAND_MAX / 2 ) ) return false;
         case PROTMESSID_CLM_REQ_CONN_CLIENTS_LIST:
             bRet = EvaluateCLReqConnClientsListMes ( InetAddr );
             break;
+
+        case PROTMESSID_CLM_REQ_CHANNEL_LEVEL_LIST:
```

Comment:
I am missing the new protocol description in the header of this file. All other protocol messages are described there.

Reply:
They were in the same commit with this code, dfb3ea7:
```
diff --git a/src/protocol.h b/src/protocol.h
index 59ed83a..e0c17f4 100755
--- a/src/protocol.h
+++ b/src/protocol.h
@@ -71,6 +71,8 @@
 #define PROTMESSID_CLM_REQ_VERSION_AND_OS     1012 // request version number and operating system
 #define PROTMESSID_CLM_CONN_CLIENTS_LIST      1013 // channel infos for connected clients
 #define PROTMESSID_CLM_REQ_CONN_CLIENTS_LIST  1014 // request the connected clients list
+#define PROTMESSID_CLM_REQ_CHANNEL_LEVEL_LIST 1015 // request the channel level list
+#define PROTMESSID_CLM_CHANNEL_LEVEL_LIST     1016 // channel level list

 // lengths of message as defined in protocol.cpp file
 #define MESS_HEADER_LENGTH_BYTE         7 // TAG (2), ID (2), cnt (1), length (2)
```

Reply:
Sorry, I was a bit unclear. I meant something like this: https://github.com/corrados/jamulus/blob/master/src/protocol.cpp#L67

Reply:
Aha!  I've not seen that before (not having needed it).  I'll have a go.